subcommands/version/cmd.go

```diff
@@ -34,6 +35,7 @@ func doVersion(cmd *cobra.Command, args []string) {
 		}
 		if err := update.Do(); err != nil {
 			if os.IsPermission(err) {
+				logrus.Debugf("Error was: %s", err)
```

Comment:
I _believe_ this belongs to the previous commit.

Reply:
i removed this an embeded it in the error message below